static/src/javascripts/projects/commercial/modules/dfp/define-slot.js

```diff
@@ -96,6 +96,56 @@ const defineSlot = (adSlotNode: Element, sizes: Object): Object => {
         slotReady = setHighMerchSlotTargeting(slot, slotTarget);
     }
 
+    if (config.switches.iasOptimisation) {
+        console.log(`Defining ${id}`);
```

Comment:
remove console.log

Reply:
As discussed, these are being left in to help better troubleshoot this problem on CODE.